tools/postgres/postgres_fixture.ts

```diff
@@ -0,0 +1,61 @@
+import { execSync } from "child_process";
+import * as dbadapters from "df/api/dbadapters";
+import { sleepUntil } from "df/common/promises";
+import { IHookHandler } from "df/testing";
+
+const USE_CLOUD_BUILD_NETWORK = !!process.env.USE_CLOUD_BUILD_NETWORK;
```

Comment:
Is there a `.env` file set up that's needed for this flag?

Reply:
no - by default it's unset, and thus `USE_CLOUD_BUILD_NETWORK` will evaluate false. cloud build uses the `bazel_test` script, which sets it, thus `USE_CLOUD_BUILD_NETWORK` evaluates true.